swssconfig/sample/sample.json

```diff
@@ -51,4 +51,14 @@
 			},
 			"OP": "SET"
 		}
-	]
\ No newline at end of file
+		{
+			"QOS_TABLE:TUNNEL_DECAP_TABLE:NETBOUNCER" : {
+				"tunnel_type":"IPINIP",
+				"dst_ip":"127.0.0.1",
```

Comment:
use another ip, e.g., 10.0.0.1

127.0.0.1 is local loopback address, should never appear on the wire whereas the decap is to decap the packet on the wire.